test/test_distributions.py

```diff
@@ -1209,23 +1249,99 @@ def test_laplace_shape_tensor_params(self):
 
 class TestKL(TestCase):
     def setUp(self):
-        self.examples = [
-            (Gamma(1, 2), Gamma(3, 4)),
+        self.finite_examples = [
+            (Bernoulli(0.7), Bernoulli(0.3)),
+            (Beta(1, 2), Beta(3, 4)),
+            (Beta(1, 2), Chi2(3)),
+            (Beta(1, 2), Exponential(3)),
+            (Beta(1, 2), Gamma(3, 4)),
+            (Beta(1, 2), Normal(-3, 4)),
             (Chi2(2), Chi2(3)),
-            (Gamma(1, 2), Chi2(3)),
             (Chi2(2), Gamma(3, 4)),
+            (Chi2(2), Exponential(3)),
+            (Dirichlet(torch.Tensor([1, 2])), Dirichlet(torch.Tensor([3, 4]))),
+            (Exponential(1), Chi2(2)),
+            (Exponential(1), Exponential(2)),
+            (Exponential(1), Gamma(2, 3)),
+            (Exponential(1), Gumbel(-2, 3)),
+            (Exponential(2), Normal(-3, 4)),
+            (Gamma(1, 2), Chi2(3)),
+            (Gamma(1, 2), Exponential(3)),
+            (Gamma(1, 2), Gamma(3, 4)),
+            (Gamma(1, 2), Gumbel(-3, 4)),
+            (Gumbel(-1, 2), Gumbel(-3, 4)),
+            (Gumbel(-1, 2), Normal(-3, 4)),  # This case fails for n = 22000
+            (Laplace(1, 2), Laplace(-3, 4)),
+            (Laplace(-1, 2), Normal(-3, 4)),
+            (Normal(-1, 2), Gumbel(-3, 4)),
+            # (Normal(-1, 2), Laplace(-3, 4)),  This case fails
```

Comment:
@fritzo I tried my best to arrive at an answer for this. Could you tell me what should be done here? If nothing works out, should I remove `_kl_normal_laplace` from the set of KL-Divergences?